### ISSUE TITLE
I seams that header total and page varaibles have changed

### ISSUE THREAD

**Issue by orukaz** [Codexshaper/laravel-woocommerce#62]
**Old conf:**

WOOCOMMERCE_WP_HEADER_TOTAL="X-WP-Total"
WOOCOMMERCE_WP_HEADER_TOTAL_PAGES="X-WP-TotalPages"

**New conf:**

WOOCOMMERCE_WP_HEADER_TOTAL="x-wp-total"
WOOCOMMERCE_WP_HEADER_TOTAL_PAGES="x-wp-totalpages"

**Comment by Codexshaper** [comment]
If it changed by original package then you can use it like above.

**Comment by orukaz** [comment]
Yep, I configured like that and I am using like that. Maybe we or I should make pull request in default config file, to change these deafult variables? Then the package works out of the box.

**Comment by Codexshaper** [comment]
I think there isn't need to change the default functionality but I'll check deeply and update it in next release.
